OJP/OJP_Trips.xsd

```diff
@@ -1699,4 +1699,78 @@
 		</xs:restriction>
 	</xs:simpleType>
 	<xs:element name="TripProblemType" type="TripProblemTypeEnumeration" substitutionGroup="ErrorType"/>
+	<xs:annotation>
+		<xs:documentation>========================================== TripChangeRequest definitions ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:group name="TripChangeRequestGroup">
+		<xs:annotation>
+			<xs:documentation>Trip change request structure.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="ChangeParams" type="TripChangeParamStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Options to control the change</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="TripResult" type="TripResultStructure">
+				<xs:annotation>
+					<xs:documentation>The trip result to be changed by the server.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="TripResponseContext" type="ResponseContextStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Context to hold objects, which are referenced within the response.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:group>
+	<xs:complexType name="TripChangeParamStructure">
+		<xs:annotation>
+			<xs:documentation>Trip change request parameter structure.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="ChangeLegRef" type="LegObjectIdType">
+				<xs:annotation>
+					<xs:documentation>Refers to the leg to be changed by the server.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="EarlierArrivalLaterDeparture" default="laterDeparture" minOccurs="0" maxOccurs="1">
+				<xs:annotation/>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="laterDeparture"/>
+						<xs:enumeration value="earlierArrival"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+			<xs:group ref="TripContentFilterGroup"/>
+			<xs:element name="FareParam" type="FareParamStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Parameters for fare calculation. Only used if IncludeFare is set (TripContentFilterGroup).</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Extension" type="xs:anyType" minOccurs="0"/>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="EarlierArrivalLaterDepartureType"/>
```

Comment:
empty element. What does it do.

Reply:
I fixed EarlierArrivalLaterDepartureType and added annotations.